one/src/lib.rs

```diff
@@ -93,6 +94,53 @@ impl Network {
         .collect::<Result<Vec<Multiaddr>, multiaddr::Error>>()
         .expect("hard coded bootstrap addresses should parse")
     }
+
+    /// Return the default ethereum rpc providers for each network.
+    pub fn default_rpc_urls(&self) -> Result<Vec<String>> {
+        match self {
+            Network::Mainnet => {
+                anyhow::bail!("no Ethereum RPC URLs specified for Mainnet")
+            }
+            Network::TestnetClay => {
+                info!("no Ethereum RPC URLs specified for Clay Testnet, defaulting to https://gnosis-rpc.publicnode.com");
+                Ok(vec!["https://gnosis-rpc.publicnode.com".to_string()])
+            }
+            Network::DevUnstable => {
+                info!("no Ethereum RPC URLs specified for dev-unstable network, defaulting to https://ethereum-sepolia-rpc.publicnode.com");
+                Ok(vec![
+                    "https://ethereum-sepolia-rpc.publicnode.com".to_string()
+                ])
+            }
+            Network::Local => {
+                info!(
+                "using default Ganache Ethereum RPC URL for Local network: http://localhost:7545"
+            );
+                // Default Ganache port
+                Ok(vec!["http://localhost:8545".to_string()])
```

Comment:
I think this is failing CI because we have to address by container name. So we probably need to update the tests to use http://ganache:8545... a bit of a chicken and egg situation with the cli flag but we should be able to use the env var for now and it will be ignored before this is in.